pontoon/base/static/js/tour.js

```diff
@@ -2,6 +2,10 @@
 //This tutorial introduces the Sideshow basics to the newcomer
 var is_entity_clicked = false;
 var is_submit_clicked = false;
+
+//TypeError: Pontoon.user.canTranslate is not a function
+var can_translate = Pontoon.user.canTranslate();
```

Comment:
I assume that's a concurrence issue. When you try to use `Pontoon.user`, it hasn't yet been correctly built. That happens when you execute JavaScript like this directly when the JS file is parsed. You'll want to enclose this in a function called when the document is finished loading. 

With jQuery you can do: 

```js
$(function () {
  var canTranslate = ...;
});
```

That code will be executed only once the document has finished loading, meaning all JS files have been downloaded and parsed.